### DISCUSSION TITLE
sp-number-field will have an infinite loop on win11 system

### DISCUSSION THREAD

**Discussion by kernelKun886** [adobe/spectrum-web-components#2590] [Q&A]
It is normal for me to test on mac and win10, but when using cep on win11, it will fall into an infinite loop, refreshing the page all the time, but it is normal on the local browser. It stands to reason that js is cross-platform, how can it appear? What about this situation? Do you have the same situation as me? My English is not good, I can only use a translator, please forgive me, thank you,

**Comment by kernelKun886** [comment]
````html
 <sp-theme color="darkest" scale="large">
    <div id="example">
    
      <sp-number-field value="10"></sp-number-field>

    </div>
  </sp-theme>
````

**Reply by Westbrook** [reply]
I don't have much experience with CEP. Could you give a little more information as to how you're leveraging this and the experience  you're seeing?

**Reply by kernelKun886** [reply]
Thanks a lot for your answer.Spectrum Web Components  is a very good library, I tried to use it to make cep extensions, using the vue framework, vue2, vue3 I have tried, other components did not find problems, but the sp-number-field component in some computers There will be a crash on the page. I have not received any logs, and the page will be reloaded infinitely. If the sp-number-field component does not assign parameters such as value, max, min, etc., there will be no crash. If it is assigned, or click up and down The button will crash, by the way, there is another point that you can't input the value with the keyboard, you can only click the up and down arrows to operate the parameters, but all this is normal in the local browser, Google, Firefox, but not in cep , the version of cep is 9.0, the software used for 10.0, 11.0 is PR&AE2020, 2021, 2022, the same software version, but there will be bugs on win11, other systems will not. I can send the github link if needed.

**Reply by kernelKun886** [reply]
"@spectrum-web-components/number-field": "^0.4.2-devmode2.0",
node_modules/@spectrum-web-components/number-field/src/NumberField.js
I found this one. If so, it is compatible. It seems to be the type of attribute.
````js
 update(changes) {
    if (changes.has("formatOptions") || changes.has("resolvedLanguage")) {
      this.clearNumberFormatterCache();
    }
    // if (changes.has("value") || changes.has("max") || changes.has("min")) {
    //   const value = this.numberParser.parse(this.formattedValue.replace(this._forcedUnit, ""));
    //   this.value = value;
    // }
    this.min=Number(this.min)
    this.max=Number(this.max)
    super.update(changes);
  }
````
